docs/documentation/siddhi-quckstart-4.0.md

```diff
@@ -1,3 +1,112 @@
+  ```
+  This Siddhi query stores incoming events for 500 milliseconds, groups them by symbol and calculates the sum for price and volume. Then it inserts the results into a stream named `outputStream`.
+  
+#### Step 2: Creating Siddhi Application Runtime
+A Siddhi application is a self contained, valid set of stream definitions and queries. This step involves creating a runtime representation of a Siddhi application by combining the stream definition and the Siddhi query you created in Step 1.
+```java
+
+SiddhiManager siddhiManager = new SiddhiManager();
+ 
+SiddhiAppRuntime executionPlanRuntime = siddhiManager.createSiddhiAppRuntime(definition + query);
+```
+In the above example, `definition + query` forms the Siddhi application.  The Siddhi Manager parses the Siddhi application and provides you with an Siddhi application runtime. This Siddhi application runtime is used to add callbacks and input handlers to the Siddhi application.
+
+#### Step 3: Registering a Callback
+You can register a callback to the Siddhi application runtime in order to receive the results once the events are processed. There are two types of callbacks:
+
++ **Query callback**: This subscribes to a query.
++ **Stream callback**: This subscribes to an event stream.
+In this example, a query callback is added because the Maven project has only one query.
+
+```java
+executionPlanRuntime.addCallback("query1", new QueryCallback() {
+     @Override
+     public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
+       EventPrinter.print(timeStamp, inEvents, removeEvents);
+     }
+});
+```
+Here, a new query callback is added to a query named `query1`. Once the results are generated, they are sent to the receive method of this callback. An event printer is added inside this callback to print the incoming events for demonstration purposes.
+
+#### Step 4: Sending Events
+In order to send events from the event stream to the query, you need to obtain an input handler as follows:
+```java
+InputHandler inputHandler = executionPlanRuntime.getInputHandler("cseEventStream");
+```
+Use the following code to start the execution plan runtime and send events:
+```java
+executionPlanRuntime.start();
+ 
+inputHandler.send(new Object[]{"ABC", 700f, 100l});
+inputHandler.send(new Object[]{"WSO2", 60.5f, 200l});
+inputHandler.send(new Object[]{"DEF", 700f, 100l});
+inputHandler.send(new Object[]{"ABC", 700f, 100l});
+inputHandler.send(new Object[]{"WSO2", 60.5f, 200l});
+inputHandler.send(new Object[]{"DEF", 700f, 100l});
+inputHandler.send(new Object[]{"ABC", 700f, 100l});
+inputHandler.send(new Object[]{"WSO2", 60.5f, 200l});
+inputHandler.send(new Object[]{"DEF", 700f, 100l});
+ 
+executionPlanRuntime.shutdown();
+```
+When the events are sent, they are printed by the event printer.
+
+For code examples, see [quick start samples for Siddhi] (https://github.com/wso2/siddhi/tree/master/modules/siddhi-samples/quick-start-samples).
```

Comment:
I think this should be quick start samples for Siddhi [https://github.com/wso2/siddhi/tree/master/modules/siddhi-samples/quick-start-samples].